starboard/tools/api_leak_detector/stub/debug/gn_built_docker_debian11_manifest

```diff
@@ -50,6 +50,7 @@ __wcscoll_l
 __wcsftime_l
 __wcsxfrm_l
 __wctype_l
+__xstat
```

Comment:
@y4vor should this symbol be added to all of the 12 manifests or just these 4?

Reply:
I guess it would be safer to add to all the stub manifests. The `__xstat` is implementation detail for the environment/OS though. So unless you actually run on Debian 10 you wouldn't know if it is needed.

Reply:
I can add it to the other ones; I guess we only run against debian11 for the tests, which is why I added to these ones.